### DISCUSSION TITLE
webpage is a mess after configurate via Github

### DISCUSSION THREAD

**Discussion by yongtaoliu** [executablebooks/meta#1055] [Q&A]
When I publish my jupyter-book through a github repository, the page shows as a mess. Does anyone know how to fix it? Thanks,.

**Comment by jan-david-fischbach** [comment]
Could you provide an example? I am not sure what exactly is messed up?

**Reply by yongtaoliu** [reply]
Thank you Jan David Black, I solved this this problem. It is because that the .nojekyII was missing
